functions/Get-DbaAgentJob.ps1

```diff
@@ -19,8 +19,8 @@ FUNCTION Get-DbaAgentJob {
 		.PARAMETER ExcludeJob
 			The job(s) to exclude - this list is auto populated from the server.
 
-		.PARAMETER Enabled
-			True will return only enabled jobs, false will return disabled jobs.
+		.PARAMETER IncludeDisabled
+			Default is to only return enabled jobs, switch will also return disabled jobs.
 
```

Comment:
I would say note the default output is only enabled jobs in the `.DESCRIPTION`. You can then keep the definition of this parameter to the point, something like: `Switch will include the disabled jobs in output.`